ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -6,101 +6,131 @@
 ### Preparation of ZTP GIT repository
 Create a GIT repository for hosting site configuration data. The ZTP pipeline will require read access to this repository.
 1. Create a directory structure with separate paths for SiteConfig and PolicyGenTemplate CRs
-1. Add pre-sync.yaml and post-sync.yaml from resource-hook-example/*policygentemplates*/ to the path for PolicyGenTemplate CRs
-1. Add pre-sync.yaml and post-sync.yaml from resource-hook-example/*siteconfig*/ to the path for SiteConfg CRs
-    > If your hub cluster operates in a disconnected environment you must update the `image` all four pre/post sync hook CRs.
+2. Export the argocd directroy from the ztp-site-generator container image by executing the following commands
+```
+    $ mkdir -p ./out
+    $ podman create -ti --name ztp-site-gen ztp-site-generator:latest bash
+    $ podman cp ztp-site-gen:/home/ztp ./out
+    $ podman rm -f ztp-site-gen
```

Comment:
Not necessary in this PR, but should we consider an easier way?  Maybe something like:
```
podman run -v out:/out ztp-site-generator:$tag extract deployment /out
```

Where `extract` is a script we provide inside the container that copies the relevant data to the given directory?